### ISSUE TITLE
fails lint

### ISSUE THREAD

**Issue by johnpapa** [angular/angular-cli#597]
ran `ng new app4` and then ran `ng lint` and got this

using latest in master.

```
src/app/app4.component.spec.ts[17, 30]: trailing whitespace
src/app/app4.component.spec.ts[18, 68]: trailing whitespace
src/app/app4.component.spec.ts[19, 30]: trailing whitespace
src/app/app4.component.spec.ts[20, 7]: trailing whitespace
src/app/app4.component.spec.ts[2, 3]: unused variable: 'async'
src/app/app4.component.spec.ts[5, 3]: unused variable: 'ddescribe'
src/app/app4.component.spec.ts[7, 3]: unused variable: 'iit'
src/app/app4.component.spec.ts[10, 3]: unused variable: 'ComponentFixture'
src/app/app4.component.spec.ts[11, 3]: unused variable: 'TestComponentBuilder'
```

cc @Brocco

**Comment by Brocco** [comment]
I want to have those unused vars there because very helpful for testing, rather than having the devs search for them.

Is there an option to turn off linting for `*.spec.ts`?

**Comment by johnpapa** [comment]
we dont want to turn off linting for specs, because that is useful too.

i agree the unused variable there are fine to show

options:
1. perhaps leave them commented out for people to add as needed?
2. remove them

I think either way we need lint to pass

**Comment by Brocco** [comment]
I can comment them out... I really dislike having to search for those things over and over again, nice to have right there at my fingertips

**Comment by johnpapa** [comment]
i agree. i never remember them all. i think comments are fine here.

**Comment by alexandermikuta** [comment]
When generating a new component tslint says:
[tslint] file should end wit a new line

the problem occurs in the index.ts-file of a component. all other .ts-files are created with a new line at the end of file.

**Comment by Brocco** [comment]
Thanks @alexandermikuta that linting issue has been resolved with PR #588 and will be part of the next release.

**Comment by hansl** [comment]
This should be fixed actually. Closing.
